### ISSUE TITLE
Taxonomic Hierarchy on Objects

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-frontend#152]
Note: this is on hold pending further discussion with YPM.
--------
**Problem Description:**
Object Type section needs to list the linked hierarchy of the type instead of only listing the linked type. 

For example, 'Plant Specimens' is listed in object types. When you select the link it lists the hierarchy on the top of the Plant Speciments record. This information should be added as linked hierarchy to the object types so users can access it there rather than selecting on the type to view hierarchy.  

**Expected Behavior/Solution:**
Object Type section will list a linked hierarchy of the type. 

**Requirements:**
- Add the taxonomic hierarchy to Object types (specimens) when available

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**
https://lux-front-tst.collections.yale.edu/view/object/00c21564-de20-427f-b66f-0a109d550594
should have linked hierarchy which is included in the type record:  
https://lux-front-tst.collections.yale.edu/view/concept/47ab8e09-e39c-46f1-b140-bdcabfbf2e1e

- Object Types: https://lux-front-tst.collections.yale.edu/view/object/2563fb25-487b-4d32-af6a-e5a0dbd06e09

**Dependencies/Blocks:**

- Blocked By: YPM work 

**Related Github Issues:**

**Related links:**
BugHerds: https://www.bugherd.com/projects/284041/tasks/2054

POLLUX Bugherds
1. https://www.bugherd.com/t/u3GmKSDG999eZG7KeCX2yw
2. https://www.bugherd.com/t/wJzQKKa4Xa3wLrurno3i7Q
3. https://www.bugherd.com/t/BSDg0XBPlGX6Msg75MkPtg
4. https://www.bugherd.com/t/3iJNs8QdFf4o5vXHWx1h8Q

**Wireframe/Mockup:**
Place wireframe/mockup for the proposed solution at end of ticket.

**Comment by roamye** [comment]
@kkdavis14 @prowns - 

Should this still be on hold by YPM work? Or can this ticket be moved forward with?

**Comment by kkdavis14** [comment]
I don't know what YPM work this is referring to. Definitely the hierarchies are, in YPM's opinion, incorrect a lot of the time, so giving them further visibility on the Object page might not be a good thing. I mentioned to Gary last week I could use his help in trying to de-tangle potentially incorrect reconciliations, but this will run into us preferring YPM's opinion > everyone elses. 

Some questions:
Will this only be for YPM objects (if so, how?)? Wouldn't YCBA or YUAG objects also get the Type hierarchy on the Object page--see screenshot? Is this just for categorized as?

![Screenshot 2024-06-24 at 4 12 11 PM](https://github.com/project-lux/lux-frontend/assets/62392285/0dbc3db5-d9ba-4d12-be96-4846c5aa55e8)

**Comment by roamye** [comment]
@kkdavis14 - the hierarchy would be for specimens but If we add the hierarchy for specimens and not other object types it can cause inconsistency. I will bring this up and your questions to UAT.

**Comment by jffcamp** [comment]
Requires discussion with YPM stakeholders
Will then need to go to data for review

**Comment by roamye** [comment]
Need to discuss this with @dcheather in next UX/BA meeting.

**Comment by roamye** [comment]
From IT Team Meeting 08/02: 
There has not been any update from YPM. This could be related to the YPM data slicing. Next time there is a meeting with YPM this will be brought up, otherwise it is up for Data to decide how we want to pursue this issue.

cc: @prowns

**Comment by roamye** [comment]
9/10 TF: 
- not in the data. will stay in forming until YPM gets into their portal process.

**Comment by zachjschwartz** [comment]
TF 7/16: Need to discuss with Ben.

**Comment by zachjschwartz** [comment]
Sarah says we can archive this ticket because several other tickets will be spinning out of the data work Ben is currently doing and it's not worth it to retrofit. Notes below.

Discussion with ben 7/16: Shared mocks. Taxonomic hierarchy will display with common names (wireframes to be added).

Example of new data: "Held or Supported by", e.g. "Held in Jar, Supported by Alcohol". (Jeff: Like Olives)

Right now the taxonomic hierarchy is comma separated list under notes (not structured, not good).

_How best to communicate changes needed in LUX to support POLLUX needs?_ Sarah thinks meeting with Andy plus Jeff, Sarah, Peter, Rob plus Kam if needed. Teams questions are fine, but probably require detailed discussions. This is chiefly a UX concern.

Peabody has extremely high number of holotypes, and might not get supported in LUX (and therefore not in POLLUX).

Data mapping work right now entails a lot of notes field info becoming structured data.

If hierarchy is important part of collections object, LUX will need to be able to store and communicate that info to a user.

Once mapping is finished, we can discuss what changes are coming down the pipe.

FIRST convo about this is probably Sarah, Jeff, Ben, Rob.

Ok to keep data in notes until there's a structured way to display on front-end, even if the structured data is present in LUX and not displaying.

Sarah will set up time in August for the POLLUX team to discuss.
